configure

```diff
@@ -11069,15 +11069,15 @@ else
   ac_have_flow_disp_h=no
 fi
 
-as_ac_File=`$as_echo "ac_cv_file_$srcdir/include/click/flowdispatcherparser.hh" | $as_tr_sh`
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for $srcdir/include/click/flowdispatcherparser.hh" >&5
-$as_echo_n "checking for $srcdir/include/click/flowdispatcherparser.hh... " >&6; }
+as_ac_File=`$as_echo "ac_cv_file_$srcdir/include/click/flowparserparser.hh" | $as_tr_sh`
```

Comment:
This is funny :p You could rename it to flowparsersquare :p
This is why FlowRuleManager or NICRuleManager is more relevant. For now , leave it as it is and we will merge properly into master

Reply:
That's a mistake actually :p flowparserparser.hh does not exist ^^